### ISSUE TITLE
Is it true that the workspaces core plugin must be activated for saving to occur?

### ISSUE THREAD

**Issue by jsmorabito** [nothingislost/obsidian-workspaces-plus#50]
If so I can add a disclaimer to the readme. I just noticed on a test vault that if the workspaces plugin is not activated then workspaces do not show up in the quick picker or modal after reopening obsidian.

**Comment by nothingislost** [comment]
Yeah, the core workspace plugin being enabled it a requirement. I could make the entire plugin just not load, or show an error in the status bar if it detects that the workspaces plugin is not missing. Open to suggestions on what the fallback behavior looks like.

**Comment by jsmorabito** [comment]
Is it possible to make it that activating workspaces plus activates the core plugin?

**Comment by nothingislost** [comment]
Nope. Even it was, that's an overreach for a plugin. We can't go mucking around with other settings that we don't own.

**Comment by jsmorabito** [comment]
> Nope. Even it was, that's an overreach for a plugin. We can't go mucking around with other settings that we don't own.

ah good point! I'm not sure of an elegant solution yet, but we can start by adding a caution disclaimer to the settings menu that shows if the core plugin isn't activated:

⚠️ Please activate the Workspaces plugin in the Core Plugins settings menu

**Comment by jsmorabito** [comment]
> I just spent 30 minutes dealing with workspaces not saving cause my core plugin wasn't enabled. It needs to be extremely obvious that it should be enabled when using this plugin.

hey @aus10code I'm very sorry to hear that. thank you for echoing this concern. There is a popup which notifies the user in the upcoming update and it can't be missed, so users will not encounter this problem in the future. Thanks again for commenting. If you have any other feedback or suggestions feel free to share more here in GitHub or with this survey: https://airtable.com/shr94ZN7tfQt6tv5n

Have a good one!
